go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -2128,23 +2130,18 @@ func commonFlows(hostSubnets []*net.IPNet, bridge *bridgeConfiguration) ([]strin
 	return dftFlows, nil
 }
 
-// ovnToHostNetworkNormalActionFlows returns the flows that allow IP{v4,v6} traffic from the OVN network to the host network
-// when the destination is on the same node as the sender. This is necessary for pods in the default network to reach
-// localnet pods on the same node, when the localnet is mapped to breth0. The expected srcMAC is the MAC address of breth0
-// and the expected hostSubnets is the host subnets found on the node primary interface.
-func ovnToHostNetworkNormalActionFlows(netConfig *bridgeUDNConfiguration, srcMAC string, hostSubnets []*net.IPNet, isV6 bool) []string {
-	var inPort, ctMark, ipFamily, ipFamilyDest string
+// hostNetworkNormalActionFlows returns the flows that allow IP{v4,v6} traffic:
+// a. from the OVN network to the host network
+// b. from the host to itself (through breth0)
```

Comment:
i don't know what this case is. If a host is trying to talk to itself, it would never go out an interface. Can you help me understand case b?

The original PR 5041 was for allowing a pod on a default network to reach a localnet pod. Which I think is what you mean by case a? If so, then I would expect case b to be allowing localnet to reach host. Is that right? If so, that isn't host to host, that is localnet pod to host.

Reply:
Ah I stretched the terminology a bit too much. This is traffic flow is for localnet to host, but it applies to everything that comes from `inPort=LOCAL`: OVN->localnet when on local gw mode, localnet->host on both gw modes.